README.md

```diff
@@ -1 +1,11 @@
 # code_challenges
+## CodeWars
+### Cakes (7kyu)
+   * Module: `code_challenges/code_wars/cakes.py`
+   * Tests:`tests/code_wars/test_cakes.py`
+   * URL: [challenge url](https://www.codewars.com/kata/525c65e51bf619685c000059/python)
+  ---
```

Comment:
I'd remove these '---' , but it's only personal preference. I don't readily see a reason this section should be broken up, but as you like.